### PR TITLE
adding missing peer port and beacon port (introduced by Patch #4)

### PR DESCRIPTION
**Pull request by m4r3x**
First of all - thank you for this repository - setting up Mordhau server was piece of cake using it.

I ran into problem with not being able to connect to server. It was fixed by adding those two ports.

Beacon port was introduced quite recently: https://steamcommunity.com/games/629760/announcements/detail/1617266961416483617

Not sure about how/where peer port is used - but adding it helped.

Cheers

### CODE REVIEW

**Comment by Hermsi1337** [comment]
Hey @m4r3x ,
I'm very glad that you like my work.

I totally missed, that there is an additional port needed for connecting to Mordhau.   
Many thanks for pointing that out!

Would you mind to also add all ports as  `EXPOSE` in the `Dockerfile`?
E.g.: https://github.com/Hermsi1337/docker-ark-server/blob/master/Dockerfile#L56

Thank you!

File: README.md [hunk 1]
```diff
@@ -49,9 +49,13 @@ services:
...
       - MAX_PLAYERS=${MAX_PLAYERS}
     ports:
-      # Port for connections from game client
+      # Game client port:
       - "7777:7777/udp"
-      # Steam's server-list port
+      # Peer port:
+      - "7778:7778/udp"
+      # Beacon port:
+      - "15000:15000"
```
Comment (Hermsi1337) [inline README.md]: Could you mark the Port as UDP?

`"15000:15000/udp"`

File: deploy/docker-compose.yml [hunk 2]
```diff
@@ -20,9 +20,13 @@ services:
...
       - MAX_PLAYERS=${MAX_PLAYERS}
     ports:
-      # Port for connections from game client
+      # Game client port:
       - "7777:7777/udp"
-      # Steam's server-list port
+      # Peer port:
+      - "7778:7778/udp"
+      # Beacon port:
+      - "15000:15000"
```
Comment (Hermsi1337) [inline deploy/docker-compose.yml]: see above

**Comment by m4r3x** [comment]
@Hermsi1337 Both suggestions are done.

**Pull request merged**